src/deploy/security/security.tf

```diff
@@ -0,0 +1,30 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+# creates rule in default security group
+resource "aws_security_group_rule" "allow_ssh" {
+  type            = "ingress"
+  from_port       = 22
+  to_port         = 22
+  protocol        = "tcp"
+  cidr_blocks = ["205.206.87.17/32"]
```

Comment:
Restricts SSH to my own public IP address. The README describes how to change this.